### ISSUE TITLE
PHP code testing isset($_POST) reports unreachable code

### ISSUE THREAD

**Issue by diddledan** [peachpiecompiler/peachpie#426]
The following code when compiled reports a warning of `PHP5011 Unreachable code detected`. Code similar to this is in WordPress. The correct behaviour is to return `true` to the `isset` when the request contains data sent to the server via HTTP POST and false in all other cases.

```php
<?php
if ( isset( $_POST ) ) {
	echo '$_POST is set'; // This is reported as PHP5011: Unreachable code detected
} else {
	echo '$_POST is not set';
}
```

A compilable test case is available at https://github.com/diddledan/peachpie-testcase-isset-dollar_POST

**Comment by jakubmisek** [comment]
Thank you @diddledan !
